packages/liveblocks-node/test-d/client.test-d.ts

```diff
@@ -1,15 +1,57 @@
 import { expectType } from "tsd";
 import { Liveblocks } from "../src/client";
-import type { PlainLsonObject, JsonObject } from "@liveblocks/core";
+import type {
+  CommentReaction,
+  CommentBody,
+  CommentBodyBlockElement,
+  CommentData,
+} from "@liveblocks/core";
 
 async () => {
   const client = new Liveblocks({ secret: "sk_xxx" });
 
-  expectType<PlainLsonObject>(await client.getStorageDocument("document-id"));
-  expectType<PlainLsonObject>(
-    await client.getStorageDocument("document-id", "plain-lson")
-  );
-  expectType<JsonObject>(
-    await client.getStorageDocument("document-id", "json")
-  );
+  const comment = await client.getComment({
+    roomId: "room-id",
+    threadId: "th_threadId",
+    commentId: "cm_commentId",
+  });
+  expectType<"comment">(comment.type);
+  expectType<string>(comment.id);
+  expectType<string>(comment.threadId);
+  expectType<string>(comment.roomId);
+  expectType<string>(comment.userId);
+  expectType<Date>(comment.createdAt);
+  expectType<Date | undefined>(comment.editedAt);
+  expectType<CommentReaction[]>(comment.reactions);
+  expectType<Date | undefined>(comment.deletedAt);
+
+  expectType<CommentBody | undefined>(comment.body);
+  expectType<1 | undefined>(comment.body?.version);
+  expectType<CommentBodyBlockElement[] | undefined>(comment.body?.content);
+
+  const thread = await client.getThread<Record<string, string>>({
+    roomId: "room-id",
+    threadId: "th_threadId",
+  });
+  expectType<"thread">(thread.type);
+  expectType<string>(thread.id);
+  expectType<string>(thread.roomId);
+  expectType<Date>(thread.createdAt);
```

Comment:
Nice! This is exactly the kind of test that will let a change inside **core** fail here in the **node** package if they're part of the public API, which may or may not be intentional! In the future, someone will be happy this test exists, I'm sure 😊